h2o-py/tests/testdir_generic_model/pyunit_generic_model_mojo_glm.py

```diff
@@ -30,18 +34,40 @@ def test(x, y, output_test, strip_part, algo_name, generic_algo_name, family):
     compare_params(glm, generic_mojo_model_from_file)
 
     output_test(original_output.getvalue(), generic_output.getvalue(), strip_part, algo_name, generic_algo_name)
-    predictions = generic_mojo_model_from_file.predict(airlines)
+
+    airlines_metrics_dataset = h2o.import_file(path=pyunit_utils.locate("smalldata/testng/airlines_train.csv"))
+    predictions = generic_mojo_model_from_file.predict(airlines_metrics_dataset)
+    metrics = generic_mojo_model_from_file.model_performance(airlines_metrics_dataset) # just loglikelihood (multiplied -1)
     assert predictions is not None
     assert predictions.nrows == 24421
     assert generic_mojo_model_from_file._model_json["output"]["model_summary"] is not None
     assert len(generic_mojo_model_from_file._model_json["output"]["model_summary"]._cell_values) > 0
     assert generic_mojo_model_from_file._model_json["output"]["variable_importances"] is not None
     assert len(generic_mojo_model_from_file._model_json["output"]["variable_importances"]._cell_values) > 0
 
+    print(generic_mojo_model_from_file._model_json["output"]["training_metrics"])
     generic_mojo_filename = tempfile.mkdtemp("zip", "genericMojo");
     generic_mojo_filename = generic_mojo_model_from_file.download_mojo(path=generic_mojo_filename)
     assert os.path.getsize(generic_mojo_filename) == os.path.getsize(original_model_filename)
 
+    if family != 'ordinal':  # loglikelihood calculation not available for ordinal family yet
+        glm_calc_like = H2OGeneralizedLinearEstimator(nfolds=2, family=family, max_iterations=2, calc_like=True,
+                                                      compute_p_values=(family == "gaussian"),
+                                                      remove_collinear_columns=(family == "gaussian"))
+        glm_calc_like.train(x=x, y=y, training_frame=airlines, validation_frame=airlines)
+        
+        print("glm training metrics:")
+        print(glm._model_json["output"]["training_metrics"])
+        print("glm calc like training metrics:")
+        print(glm_calc_like._model_json["output"]["training_metrics"])
+        print("metrics:")
+        print(metrics)
+        
+        assert math.isclose(glm_calc_like._model_json["output"]["training_metrics"]._metric_json["AIC"], 
+                            metrics._metric_json["AIC"], rel_tol=1e-3), "The numbers are not close enough."
+        assert math.isclose(-glm_calc_like._model_json["output"]["training_metrics"]._metric_json["loglikelihood"], 
+                            metrics._metric_json["loglikelihood"], rel_tol=1e-3), "The numbers are not close enough."
```

Comment:
Julia:  I was surpised that the results are a little different.  I was hoping they should match exactly....  Do you know the reason for the discrepancies?